packages/client/src/components/NavBar/index.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import IconButton from '@material-ui/core/IconButton';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Grid from '@material-ui/core/Grid';
+import { Link } from 'react-router-dom';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import PropTypes from 'prop-types';
+import Modal from '../Modal';
+import Wizard from '../Wizard';
+import Button from '../Button';
+import styled from 'styled-components';
+import { COLORS } from '../../constants';
+
+
+const Wiz_content = ['page1', <Button>oh boi</Button>, 'page3']
```

Comment:
what is this? Lmao

Reply:
lol I think this is for the register modal haha (which is part of Navbar)